workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+
+on:
+  workflow_call:
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Set up NodeJS
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{vars.NODE_VERSION}}
```

Comment:
Again, is this really needed? Are tree-sitter grammars usually tied down to a specific node version?